assignments/objects.js

```diff
@@ -49,16 +86,35 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name": "Susan",
+  "age": "70"
+}
 
-// Log the parent object's name
+parent.child = {
+  "name": "George",
+  "age": "50"
+}
 
-// Log the child's age
+parent.child.grandchild = {
+  "name": "Sam",
+  "age": "30"
+}
 
-// Log the name and age of the grandchild
+parent.speak = "Hello, my name is Susan";
```

Comment:
```suggestion
parent.speak = function() {return `Hello, my name is ${this.name}`};
```
Follow this for the child and granchild as well